SYSDEPS.md

```diff
@@ -1,4 +1,4 @@
-VERSION
+VERSION.md
```

Comment:
Let's go ahead and merge PR #33 in front of this PR, which would eliminate this change.

Reply:
Yes, if we're willing to promote that one to 1.07 (I am).

Reply:
I moved PR #33 to 1.08, let's leave this change in this patch.